powershell_kernel/kernel.py

```diff
@@ -51,15 +53,30 @@ def __init__(self, **kwargs):
 
     def do_execute(self, code, silent, store_history=True,
                    user_expressions=None, allow_stdin=False):
-        if not code.strip():
+        try:
+            if not code.strip():
+                return {'status': 'ok', 'execution_count': self.execution_count,
```

Comment:
So if cell is empty it'll succeed even though Powershell isn't installed? Just verifying that's the expected behavior (is that consistent with some other kernel like SparkMagic, for example)?

Reply:
Sparkmagic also does nothing if the cell is empty.